docs/tools/terraform/reference/cookbook/kafka-connect-terraform-recipe.rst

```diff
@@ -146,7 +146,7 @@ Here is the sample Terraform file to stand-up and connect all the services. Keep
       connector_name = "kafka-os-con1"
       config = {
         "topics" = aiven_kafka_topic.topic-logs-app-1.topic_name
-        "connector.class" : "io.aiven.kafka.connect.opensearch.OpensearchSinkConnector"
+        "connector.class" = "io.aiven.kafka.connect.opensearch.OpensearchSinkConnector"
```

Comment:
needs to be formatted still

Reply:
ran the tool this time rather than doing it by hand :D 

let's check the deploy preview to make sure they look OK.

Reply:
pls check again :)